.gitignore

```diff
@@ -213,3 +213,5 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
+
+.idea
```

Comment:
It's better to avoid the full exclusion of .idea folder since we save there pre-configured project info (DB connection, Debug settings, etc.)